libbeat/docs/runconfig.asciidoc

```diff
@@ -11,7 +11,7 @@
 //////////////////////////////////////////////////////////////////////////
 
 [[configuration-run-options]]
-=== Run Options (Optional)
+=== Run Options Configuration (Optional)
```

Comment:
@ruflin I don't see runoptions in the packetbeat.full.yml (or any of the other full config files). Maybe @monicasarbu knows?